keps/sig-storage/20190124-csi-topology.md

```diff
@@ -53,6 +92,15 @@ The rest of the document includes required information missing from the original
   * (Negative) Volume provisioning with delayed volume binding and pod zone missing from AllowedTopologies.
 Initially topology tests are run against a single CSI driver. As the CSI test suites become modularized they will run against arbitrary CSI drivers.
 
+### GA testing
+* Manual e2e testing for upgrade and version skew scenarios.
+  * An older driver that only understands v1beta1 should continue to work when the cluster is
```

Comment:
`driver` -> `sidecar`? Drivers do not know anything about CSINodes.

Reply:
fixed